packages/runtime-core/src/rendererTemplateRef.ts

```diff
@@ -12,7 +12,7 @@ import {
 import { isAsyncWrapper } from './apiAsyncComponent'
 import { getExposeProxy } from './component'
 import { warn } from './warning'
-import { isRef } from '@vue/reactivity'
+import {isRef, reactive} from '@vue/reactivity'
```

Comment:
Maybe you have to run `pnpm format` first.

Reply:
ok, i have formatted the code